src/home/home.scss

```diff
@@ -121,8 +125,11 @@
         line-height: unset;
       }
 
-      .subtext {
-        color: $Neutral03;
+      .noMissColumn {
```

Comment:
this is confusing because the .noMissColumn lines are already in master.  What did you do here?  Can you please clean this up wrt the master branch?

Reply:
No, i cannot find the .noMissColumn in master, I confirmed again